src/components/ADempiere/Form/VPOS/Collection/overdrawnInvoice/index.vue

```diff
@@ -0,0 +1,278 @@
+            >
+              <el-row>
+                <el-col v-for="(field, index) in fieldsList" :key="index" :span="8">
+                  <field-definition
+                    :key="field.columnName"
+                    :metadata-field="field"
+                  />
+                </el-col>
+              </el-row>
+            </el-form>
+          </div>
+        </el-card>
+      </div>
+      <div v-if="caseOrder === 2">
+        <el-card>
+          <div slot="header" class="clearfix">
+            <span> {{ $t('form.pos.collect.overdrawnInvoice.below') }} </span>
+            <el-radio v-model="option" :label="1" style="float: right; padding: 3px 0">
+              {{ $t('form.pos.collect.overdrawnInvoice.adjustDocument') }}
+            </el-radio>
+          </div>
+          <el-form label-width="120px">
+            <el-form-item>
+              <p>
+                <b> {{ $t('form.pos.collect.orderTotal') }} </b> {{ formatPrice(totalOrder, currency.iSOCode) }}
+                <el-divider direction="vertical" />
+                <b> {{ $t('form.pos.collect.totalInvoiced') }} </b> {{ formatPrice(pay, currency.iSOCode) }}
+                <el-divider direction="vertical" />
+                <b> {{ $t('form.pos.collect.pending') }} </b> {{ formatPrice(pending, currency.iSOCode) }}
+              </p>
+            </el-form-item>
+          </el-form>
+        </el-card>
+      </div>
+      <span slot="footer" class="dialog-footer">
+        <el-button
+          type="danger"
+          class="custom-button-create-bp"
+          icon="el-icon-close"
+          @click="close"
+        />
+        <el-button
+          type="primary"
+          class="custom-button-create-bp"
+          icon="el-icon-check"
+          @click="success"
+        />
+      </span>
+    </el-dialog>
+  </div>
+</template>
+
+<script>
+import { formatPrice } from '@/utils/ADempiere/valueFormat.js'
+import formMixin from '@/components/ADempiere/Form/formMixin'
+import posMixin from '@/components/ADempiere/Form/VPOS/posMixin.js'
+import fieldsListOverdrawnInvoice from './fieldsListOverdrawnInvoice.js'
+import { overdrawnInvoice } from '@/api/ADempiere/form/point-of-sales.js'
+
+export default {
+  name: 'OverdrawnInvoice',
+  mixins: [
+    formMixin,
+    posMixin
+  ],
+  props: {
+    change: {
+      type: Number,
+      default: 0
+    },
+    pay: {
+      type: Number,
+      default: 0
+    },
+    pending: {
+      type: Number,
+      default: 0
+    },
+    totalOrder: {
+      type: Number,
+      default: 0
+    },
+    currency: {
+      type: Object,
+      default: undefined
+    },
+    metadata: {
+      type: Object,
+      default: () => {
+        return {
+          uuid: 'OverdrawnInvoice',
+          containerUuid: 'OverdrawnInvoice'
+        }
+      }
+    }
+  },
+  data() {
+    return {
+      option: 1,
+      fieldsList: fieldsListOverdrawnInvoice
+    }
+  },
+  computed: {
+    showDialogo() {
```

Comment:
please change computed name to `showDialog` (without the `o` at the end of the Dialog)